### PR TITLE
slack app redirect

### PR DESCRIPTION
**Pull request by pythexcel**

### CODE REVIEW

File: app/config.py [hunk 1]
```diff
@@ -20,9 +20,12 @@
...
 }
 
-#bounced_mail_since = '1-Jan-2020'
-#remind_mail_since = '1-Jan-2020'
+oauth_url = 'https://slack.com/api/oauth.v2.access'
+client_id = '124720392913.862592480496'
+client_secret = '0405e4f1150a7a9dcbaa4442e3aeea4f'
+client_redirect_uri = 'http://176.9.137.77:8012/slack/redirect'
 
+slack_redirect_url = 'http://176.9.137.77:81/dashboard/jobs-overview'
```
Comment (manishiitg) [inline app/config.py:28]: what is this for?

File: app/config.py [hunk 2]
```diff
@@ -20,9 +20,12 @@
     'smtp.mail.me.com' : 1000
 }
 
-#bounced_mail_since = '1-Jan-2020'
-#remind_mail_since = '1-Jan-2020'
+oauth_url = 'https://slack.com/api/oauth.v2.access'
+client_id = '124720392913.862592480496'
+client_secret = '0405e4f1150a7a9dcbaa4442e3aeea4f'
+client_redirect_uri = 'http://176.9.137.77:8012/slack/redirect'
```
Comment (manishiitg) [inline app/config.py:26]: this/

Comment (pythexcel) [reply app/config.py:26]: sir this is the slack app credentials which are needed for autorizing slack application

File: app/config.py (hunk 1 above)
Comment (pythexcel) [reply app/config.py:28]: sir this is the url that will be redirect after installed in to the workspace and then this url will redirect to the slack app url that will hold the code

**Pull request closed without merging**